docs/dev/error-messages.md

```diff
@@ -241,6 +241,28 @@ This isn't always restricted to the tests, either; an app under test which consu
 
 Break out your long tests into shorter tests and/or make sure that your tests are not filling up a lot of disk space on the VM.
 
+
+### Unable to Find Device Within 15 Minutes
```

Comment:
Hmm, this is not the correct message, the customer will not see the 15 min warning, but the 900000ms warning. Let's use the exact same message so it's easier to find for out customers